### DISCUSSION TITLE
FTS installation Q&A

### DISCUSSION THREAD

**Discussion by brothercorvo** [FreeTAKTeam/FreeTakServer#353] [FreeTAKServer Support]
Conversation about FTS Installation

**Comment by minyoss** [comment]
Hi
I do have some failed errors while starting the FTS server as below, and would like to know why and if this can be fixed.  Despite the failed messages, the server still runs fine.  

Cheers,

**[START traces ]**
Cannot find CA file locally so generating one
Generating Key...
Generating Key...
failed to register component: /usr/local/lib/python3.10/dist-packages/digitalpy/core/component_management, with error: module 'digitalpy.core.component_management.component_management_facade' has no attribute 'ComponentManagement'
failed to register component: /usr/local/lib/python3.10/dist-packages/digitalpy/core/persistence, with error: module 'digitalpy.core.persistence.persistence_facade' has no attribute 'Persistence'
failed to register component: /usr/local/lib/python3.10/dist-packages/digitalpy/core/zmanager, with error: No module named 'Catalog'
failed to register component: /usr/local/lib/python3.10/dist-packages/digitalpy/core/util, with error: No module named 'Catalog'
failed to register component: /usr/local/lib/python3.10/dist-packages/digitalpy/core/logic, with error: No module named 'Catalog'
failed to register component: /usr/local/lib/python3.10/dist-packages/digitalpy/core/telemetry, with error: No module named 'Catalog'
failed to register component: /usr/local/lib/python3.10/dist-packages/digitalpy/core/health, with error: No module named 'Catalog'
failed to register component: /usr/local/lib/python3.10/dist-packages/digitalpy/core/translation, with error: No module named 'Catalog'
failed to register component: /usr/local/lib/python3.10/dist-packages/digitalpy/core/queries, with error: No module named 'Catalog'
failed to register component: /usr/local/lib/python3.10/dist-packages/digitalpy/core/network, with error: name 'Facade' is not defined
failed to register component: /usr/local/lib/python3.10/dist-packages/FreeTAKServer/core/fts_core, with error: module 'FreeTAKServer.core.fts_core.fts_core_facade' has no attribute 'FtsCore'
failed to register component: /usr/local/lib/python3.10/dist-packages/FreeTAKServer/core/fts_configuration, with error: module 'FreeTAKServer.core.fts_configuration.fts_configuration_facade' has no attribute 'FtsConfiguration'
failed to register component: /usr/local/lib/python3.10/dist-packages/FreeTAKServer/core/domain, with error: module 'FreeTAKServer.core.domain.domain_facade' has no attribute 'Domain'
failed to register component: /usr/local/lib/python3.10/dist-packages/FreeTAKServer/core/data_package, with error: module 'FreeTAKServer.core.data_package.data_package_facade' has no attribute 'DataPackage'
failed to register component: /usr/local/lib/python3.10/dist-packages/FreeTAKServer/components/extended/report, with error: module 'FreeTAKServer.components.extended.report.report_facade' has no attribute 'Report'
failed to register component: /usr/local/lib/python3.10/dist-packages/FreeTAKServer/components/extended/mission, with error: module 'FreeTAKServer.components.extended.mission.mission_facade' has no attribute 'Mission'
failed to register component: /usr/local/lib/python3.10/dist-packages/FreeTAKServer/components/extended/federation, with error: module 'FreeTAKServer.components.extended.federation.federation_facade' has no attribute 'Federation'
failed to register component: /usr/local/lib/python3.10/dist-packages/FreeTAKServer/components/extended/repeater, with error: module 'FreeTAKServer.components.extended.repeater.repeater_facade' has no attribute 'Repeater'
failed to register component: /usr/local/lib/python3.10/dist-packages/FreeTAKServer/components/extended/companion_parrot, with error: module 'FreeTAKServer.components.extended.companion_parrot.companion_parrot_facade' has no attribute 'CompanionParrot'
failed to register component: /usr/local/lib/python3.10/dist-packages/FreeTAKServer/components/extended/cotmanager, with error: module 'FreeTAKServer.components.extended.cotmanager.cotmanager_facade' has no attribute 'Cotmanager'
failed to register component: /usr/local/lib/python3.10/dist-packages/FreeTAKServer/components/extended/track_manager, with error: module 'FreeTAKServer.components.extended.track_manager.track_manager_facade' has no attribute 'TrackManager'
failed to register component: /usr/local/lib/python3.10/dist-packages/FreeTAKServer/components/extended/master_parrot, with error: module 'FreeTAKServer.components.extended.master_parrot.master_parrot_facade' has no attribute 'MasterParrot'
initiate_sockets
start 213
starting now
running api
0.0.0.0
8080
(5945) wsgi starting up on http://0.0.0.0:8080
start 213
starting SSL now
0.0.0.0
8443
(5952) wsgi starting up on https://0.0.0.0:8443
CoT Service Started
CoTService started
server started
started federation federate service
server started
**[END traces]**

**Comment by cgmckeever** [comment]
Ever figure this out?

**Reply by brothercorvo** [reply]
This is a minor issue with no impact that has been fixed in 2.1 Release , to be released sometime this summer
